lib/l10n_utils/extract.py

```diff
@@ -0,0 +1,61 @@
+# mostly borrowed from tower
```

Comment:
Any chance we can borrow tests for these functions from tower as well, or would you be willing to add some?